src/tools/fuzzing/fuzzing.cpp

```diff
@@ -507,14 +507,14 @@ void TranslateToFuzzReader::finalizeTable() {
   for (auto& table : wasm.tables) {
     ModuleUtils::iterTableSegments(
       wasm, table->name, [&](ElementSegment* segment) {
-        // If the offset is a global that was imported (which is ok) but no
-        // longer is (not ok) we need to change that.
-        if (auto* offset = segment->offset->dynCast<GlobalGet>()) {
-          if (!wasm.getGlobal(offset->name)->imported()) {
```

Comment:
Why did the check for `->imported()` go away? I think we only need to zero it out of there is a `global.get` that is not imported.

Reply:
Hmm, I see that `finalizeMemory` does do this check, but with a note that imported globals are never encountered. Indeed, `setupGlobals` removes all imports. I guess I can follow `finalizeMemory`'s lead here.